### DISCUSSION TITLE
Video: JSON in Godror

### DISCUSSION THREAD

**Discussion by cjbj** [godror/godror#242] [Show and tell]
As part of the Oracle Database JSON Office Hours series, @sudarshan12s and @sharadraju recorded a presentation on using JSON in Godror.  Check it out: https://asktom.oracle.com/pls/apex/asktom.search?oh=17654

**Comment by tgulacsi** [comment]
Thanks, I've tried to add it: 989bef640c0ff68dfef1a2b13411eefcc52b9f33

**Reply by sharadraju** [reply]
Hi Tamas,
The youtube Link for the JSON in Godror video is : [https://www.youtube.com/watch?v=7sJxwlKrkn8](https://www.youtube.com/watch?v=7sJxwlKrkn8)

**Comment by cjbj** [comment]
That's great. If you're keeping a list, there is this older video too: Using the Go Language for Efficient Oracle Database Applications https://www.youtube.com/watch?v=5UWtyGdf3sY
